DashAI/front/.gitignore

```diff
@@ -8,9 +8,6 @@
 # testing
 /coverage
 
-# production
-/build
-
```

Comment:
Same here, I don't think it's a good idea to pull builds from the gitignore.

Reply:
The build/ from root gitignore catches this one, it was redundant